db/WiredTiger

```diff
@@ -0,0 +1,2 @@
+WiredTiger
```

Comment:
Your db folder did not get gitignored properly. You should just put ``db`` in your .gitignore.